webpack.config.js

```diff
@@ -1,24 +1,28 @@
 const path = require('path');
 
 module.exports = {
-    mode: 'development',
-    entry: './client/index.jsx',
-    output: {
-        filename: 'main.js',
-        path: path.resolve(__dirname, 'public')
-    },
-    module: {
-        rules: [
-            {
-                test: /jsx?$/,
-                exclude: /(node_modules|bower_components)/,
-                use: {
-                    loader: 'babel-loader',
-                    options: {
-                        presets: ['@babel/preset-env', '@babel/preset-react']
-                    }
-                }
+  mode: 'development',
+  entry: './client/index.jsx',
+  output: {
+      filename: 'main.js',
+      path: path.resolve(__dirname, 'public')
+  },
+  module: {
+      rules: [
+          {
+            test: /\.css$/i,
+            use: ['style-loader', 'css-loader'],
```

Comment:
Are these two lines needed for Styled components? I don't remember seeing them in the docs